### ISSUE TITLE
Provide Branding Materials for Discourse Forum

### ISSUE THREAD

**Issue by sunnydean** [ESA-EarthCODE/EarthCODE#16]

**Comment by sunnydean** [comment]
From Anca:

"For the website: we’ve submitted a request to our graphics bureau to produce:

Website hero banner
Background image
Favicon
Variant of the key visual with transparent background and with no text

If we need to ask for more let me know.
 
For the colour palette, you can use the key visual as a reference."

**Comment by Schpidi** [comment]
I suppose this can be closed right @sunnydean?

**Comment by sunnydean** [comment]
Yes, I think so; branding materials that were used for the Discourse can be found here:

https://esait.sharepoint.com/:f:/r/sites/EarthCODE/Shared%20Documents/General/WS-3-Community/[Graphics-portal](https://esait.sharepoint.com/:f:/r/sites/EarthCODE/Shared%20Documents/General/WS-3-Community/Graphics-portal?csf=1&web=1&e=SfLV8K)?csf=1&web=1&e=SfLV8K
